api/CustomElementRegistry.json

```diff
@@ -0,0 +1,65 @@
+{
+  "api": {
+    "CustomElementRegistry": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomElementRegistry",
+        "description": "The CustomElementRegistry interface provides methods for registering custom elements and querying registered elements.",
+        "support": {
+          "chrome": {
+            "version_added": "33",
+            "notes": "Chrome 36+/Opera 20+ implemented a previous version of Custom Elements (v0) that used `.registerElement()`"
+          },
+          "firefox": {
+            "version_added": true,
+            "notes": "Enabled through the `dom.webcomponents.enabled` preference in `about:config`"
+          },
+          "safari": {
+            "version_added": "10.1",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+          "edge": {
+            "version_added": false,
+            "notes": "Under consideration"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "41",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+
+
+          "chrome_android": {
+            "version_added": "61",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+          "webview_android": {
+            "version_added": "56",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+          "edge_mobile": {
+            "version_added": false,
+            "notes": "Under consideration"
+          },
+          "safari_ios": {
+            "version_added": "10.1",
+            "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
+          },
+          "ie_mobile": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
```

Comment:
Are you sure Opera restrict this to the desktop version?

Reply:
yup @teoli2003 

Fixed in: https://github.com/mdn/browser-compat-data/pull/623/commits/b734b34f342af0c1ad68836850c5bc7ca1cae339